examples/records/records.bal

```diff
@@ -1,23 +1,32 @@
 import ballerina/io;
 
-// This defines an open record type named `Student` which in addition to the fields defined here, allows additional
-// `string` fields.
-type Student record {|
+// This defines an open record type named `Student`. The `{` and `}` delimiters indicate that this record type
```

Comment:
Should we change "This defines an " to "Define an " ?

Given that this is also the first records example, should we start with what a record is? By saying if can hold values for the particular the fields, and then talk about additional fields?

Reply:
"This defines an" is fine right? Given that we are referring to a record type descriptor by "this".

A brief explanation about what a record is given in the records.description file

Reply:
IINM, we've used "Define a ...." in general instead of saying "This ...". We can decide on a convention maybe? I personally prefer the former.

IMO it'll still be useful to start with the fields here too. How about swapping to have something like:
```
// Define an open record type named `Student`. The `{` and `}` delimiters indicate that in addition to the fields defined here
// this record type allows values with additional pure-typed (i.e., `anydata|error`) fields.
```

Reply:
+1 will change